components/ai_chat/ai_chat.mojom

```diff
@@ -11,6 +11,7 @@ enum CharacterType {
 };
 
 enum ConversationTurnVisibility {
+  DONT_ADD,
```

Comment:
If it makes sense to consider all three values together as different levels of visibility or inclusion in the chat history, then it may make sense to keep them in the same enum. However, if DONT_ADD represents a completely different concept that is not related to visibility or chat history, it may be clearer to have a separate enum or variable for it.

I have plans to eventually abstract the data model to a separate interface and have the following variants for visibility:

1. Private: Only visible to the system and added to the history
2. Public: Visible to the user and added to the history

I guess you can keep it as DONT_ADD or INTERNAL for now and maybe add a comment. I'll clean it up in a follow-up PR. As per this PR change looks like HIDDEN variant is not being used anywhere.

`DONT_ADD, // does not get added to chat history, used as a flag internally`

Reply:
Yeah, a separate enum with `Type` might be better then. I changed it to INTERNAL, I liked it better. Then you can handle the decoupling in the refactor you mentioned 👍